app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java

```diff
@@ -219,6 +229,14 @@ protected void onPreferenceChanged(SharedPreferences prefs, String key) {
                 boolean extraLaunchersEnabled = prefs.getBoolean(key, false);
                 ActivityUtils au = new ActivityUtils(getActivity());
                 au.applySpecialLaunchersVisibility(extraLaunchersEnabled);
+            } else if (eq(key, R.string.pref_key__default_encryption_password) && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT && !TextUtils.isEmpty(prefs.getString(key, null))) {
```

Comment:
Now it's seems impossible for me to delete the password, or miss I something?

Reply:
Correct, but I dont need see the need to clear password currently. Instead, changing password is less error-prone now.
For decision, wheter encrypt or not "is file encrypted" or "newfile checked encrypt" is now the point, and not "is password set && !\*\*\*"

Reply:
If you deleted the password no checkbox was shown and the app works same as now on master.

Reply:
I'm fine with checkbox showing up. The problem was that it has been auto-checked when a password was set.
I changed it to remember last check state, so OK.

Reply:
Furthermore I'm not a fan of redundant data, but it's safe. In case the password become lost we will get a NPE.

Reply:
Exactly. If somebody really wants to easy "clear" password, then they can write set new gibberish/password. Thats basically also what some SSD's own encryption do :smile: .